pkg/controller/directvolumemigration/rsync.go

```diff
@@ -584,6 +614,12 @@ func (t *Task) createRsyncClientPods() error {
 		trueBool := true
 		runAsUser := int64(0)
 
+		pvcNodeMap, err := t.getPVCNodeNameMap()
```

Comment:
This produces a list across namespaces, so this should be called outside the `for ns, vols` loop.

Reply:
Done, Updated the PR